tests/test_global.py

```diff
@@ -0,0 +1,292 @@
+# Run this before deployments, with pytest!
```

Comment:
This test file is riddles with absolute filepaths. Anyone trying to run the test would have to make a bunch of changes just to refer to the correct file locations. Could we make that part of settings somehow? That would definitely help make this easier to run.

Reply:
only cloud detection now

Reply:
You could also provide instructions on where to put the cloud_detection files within the folder structure when you download them. Then you don't have to hard code the filepaths here.

Reply:
still needs work